inspirehep/config.py

```diff
@@ -596,3 +598,16 @@
 }
 """Updates default properties that should be added to the Holding Pen index
 mappings."""
+
+CACHED_VIEWS = {
+    'single_entry': {
```

Comment:
What is `single_entry`? The detailed record with references etc? Shall we call it `detailed`?